src/javascript/binary/websocket_pages/user/account/settings/limits/limits.init.js

```diff
@@ -1,56 +1,57 @@
 const template = require('../../../../../base/utility').template;
 const Content  = require('../../../../../common_functions/content').Content;
 const addComma = require('../../../../../common_functions/string_util').addComma;
-const LimitsUI = require('./limits.ui').LimitsUI;
+const LimitsUI = require('./limits.ui');
 const localize = require('../../../../../base/localize').localize;
 const Client   = require('../../../../../base/client').Client;
 const elementTextContent  = require('../../../../../common_functions/common_functions').elementTextContent;
 const elementInnerHtml    = require('../../../../../common_functions/common_functions').elementInnerHtml;
 
-const LimitsWS = (function() {
+const Limits = (() => {
```

Comment:
Not actually an issue, but better to not use the same name as the one in `limits.js`